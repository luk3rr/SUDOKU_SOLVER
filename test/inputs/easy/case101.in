090107300 380024000 175008000 009600100 864030000 000049007 018006000 000200806 000000204 
