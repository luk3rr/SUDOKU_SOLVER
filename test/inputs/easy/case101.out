Solving the following grid:
0 9 0 | 1 0 7 | 3 0 0 
3 8 0 | 0 2 4 | 0 0 0 
1 7 5 | 0 0 8 | 0 0 0 
------+-------+------
0 0 9 | 6 0 0 | 1 0 0 
8 6 4 | 0 3 0 | 0 0 0 
0 0 0 | 0 4 9 | 0 0 7 
------+-------+------
0 1 8 | 0 0 6 | 0 0 0 
0 0 0 | 2 0 0 | 8 0 6 
0 0 0 | 0 0 0 | 2 0 4 


Solution found :')

4 9 2 | 1 6 7 | 3 8 5 
3 8 6 | 5 2 4 | 7 9 1 
1 7 5 | 3 9 8 | 4 6 2 
------+-------+------
7 3 9 | 6 5 2 | 1 4 8 
8 6 4 | 7 3 1 | 5 2 9 
5 2 1 | 8 4 9 | 6 3 7 
------+-------+------
2 1 8 | 4 7 6 | 9 5 3 
9 4 3 | 2 1 5 | 8 7 6 
6 5 7 | 9 8 3 | 2 1 4 

Algorithm: BFS
Total time: 10 ms
Total expanded states: 1708
